planet/clients/orders.py

```diff
@@ -344,27 +347,37 @@ async def poll(
         order_id: str,
         state: str = None,
         wait: int = 10,
```

Comment:
The user-facing information for the polling updates every 10 seconds. This feels somewhat clunky/slow from a UX point of view. By changing this to 1 second it feels much more smooth and somewhat more reliable, as I feel less anticipation waiting to see what's going to happen next. Is there a reason as to why the user must wait so long between updates? Perhaps not much would happen on such a short timescale (i.e., `running` > `success`), but from a UX point of view, seeing the clock tick every second feels better, or at least it does for me :)